test/src/module/command/restoreTest.c

```diff
@@ -1337,32 +1352,6 @@ testRun(void)
 
         TEST_RESULT_LOG("P00 DETAIL: databases found for selective restore (1, 12168, 16380, 16381, 16384, 16385)");
 
-        // -------------------------------------------------------------------------------------------------------------------------
-        TEST_TITLE("error on system database with non-systemId selected");
```

Comment:
Why was this test removed? Can you point me to where we are testing that --db-include when given a system database id that is not a system-id is bring tested?

Reply:
Any db in systemDbIdList will be affected. I found this was a duplicate test, but I can bring it back if you think it's best

Reply:
fixed in commit a69c81b